src/website/views/components/toolbar/_properties.pug

```diff
@@ -1,6 +1,151 @@
-h2 Web Component
-h3 Properties
-.chi-card.-align--center.-empty
-  .chi-card__content
-    .chi-card__caption
-      | Web component coming soon
+h2 Chi Vue
+h3 Props
+
+h4 Search Input
+p.-text
+  | Please visit Search input dedicated documentation page <a href="../forms/search-input/#props">here</a>
+
+h4 Filters
+section.chi-table.chi-table__options.-bordered.-my--3
+  div(style='overflow-x:auto;')
+    table.-text(cellpadding='0' cellspacing='0')
+      thead
+        tr
+          th
+            div Property
+          th
+            div Description
+          th
+            div Type
+          th
+            div Default
+      tbody
+        tr
+          td
+            div
+              code filtersData
+          td
+            div To define filters in the toolbar.
+          td
+            div
+              code DataTableFilters[]
+          td
+            div
+              code undefined
```

Comment:
The suggestion was `undefined` only.. `code unfedined` is inconsistent with properties of other components

<img width="802" alt="Screenshot 2021-07-26 at 12 47 48" src="https://user-images.githubusercontent.com/25136131/126977328-a51ecbc6-bc87-44b6-a93c-e45c3978c443.png">